subworkflows/nf-core/fastq_extract_kraken_krakentools/main.nf

```diff
@@ -0,0 +1,27 @@
+include { KRAKEN2_KRAKEN2                } from '../../../modules/nf-core/kraken2/kraken2/main'
+include { KRAKENTOOLS_EXTRACTKRAKENREADS } from '../../../modules/nf-core/krakentools/extractkrakenreads/main'
+
+workflow FASTQ_EXTRACT_KRAKEN_KRAKENTOOLS {
```

Comment:
Maybe the name should be 'FASTQ_EXTRACTREADS_KRAKEN_KRAKENTOOLS` I guess 'extract' could be like, extract genus hits from the report or something?

Reply:
Also I guess the subworkflow is doing more than extracting, it's also profiling?

Maybe `FASTQ_PROFILE_EXTRACTREADS_KRAKEN_KRAKENTOOLS`

It makes it more obvious from reading the name  guess what it does?

Reply:
While I agree with `FASTQ_EXTRACTREADS_KRAKEN_KRAKENTOOLS`, I think profiling is a bit too much. Also, the krakentools script is `extract_kraken_reads.py` - and that's what I would be looking for 🤔

Reply:
Fair enough about it being too long, I realise now `FASTQ_KRAKEN_EXTRACTREADS_KRAKENTOOLS` would be sufficient, as kraken is in there, for some reason my brain 'skipped' that be (basically initially I didn't instinctively understand where the reads were being extracted from... thus my request to add profile, but if KRAKEN is there it's clearer)